core/terraform/resource_processor/vmss_porter/cloud-config.yaml

```diff
@@ -63,6 +63,8 @@ write_files:
       LOGGING_LEVEL=${logging_level}
       ENABLE_CMK_ENCRYPTION=${enable_cmk_encryption}
       KEY_STORE_ID=${key_store_id}
+      ENABLE_MALWARE_SCANNING=${enable_malware_scanning}
```

Comment:
We generally want to pass only Resource Process params here. This is a param used in the bundles and should be passed via the rp_bundle_values. There was something we did to support named params provided in the config.yaml file to allow it to be more explicit. I will look it up.

Reply:
I take your point, but this is a system-wide setting that's used in core and _also_ needs to travel through to the workspace. Whether a workspace bundle uses that setting is dependent on the bundle?

Reply:
I take your point, but the `enable_airlock_malware_scanning` value already exists in `config.sample.yaml` with a default of `true`. Currently the`rp_bundle_values` section is commented out in here, so we would have to uncomment this and also put the `enable_malware_scanning` value in here. This could then get out of sync with the value in core.

Reply:
I have a [PR ](https://github.com/microsoft/AzureTRE/pull/4437) that will help avoid making changes in the resource processor code for cases like this.
I will reply again after it's been merged.

Reply:
Oh, can you fix the title and/or description of this PR to reflect it is targeting the outgoing airlock part (and not the internal where the malware scanner works just fine)?

Reply:
The PR is now merged. IMO you should amend the following:
1. Revert all changes to the RP code and add config.yaml settings or other you want to be available for the bundle [here](https://github.com/microsoft/AzureTRE/blob/main/core/terraform/resource_processor/vmss_porter/locals.tf#L16)
2. Update the title and/or description of the PR as commented above.
3. Consider changing the name of params you make available and use in the bundle as I'm not sure it's clear enough in the bundle context to note it's related to the airlock feature.